filebeat/input/filestream/environment_test.go

```diff
@@ -86,6 +87,19 @@ func (e *inputTestingEnvironment) mustCreateInput(config map[string]interface{})
 	return inp
 }
 
+func (e *inputTestingEnvironment) createInput(config map[string]interface{}) (v2.Input, error) {
```

Comment:
Nit: this function could be called from `mustCreateInput`:

```golang
func (e *inputTestingEnvironment) mustCreateInput(config map[string]interface{}) v2.Input {
	e.t.Helper()
	inp, err := createInput(config)
	if err != nil {
		e.t.Fatalf("failed to create input using manager: %+v", err)
	}
	return inp
}
```

Reply:
Also, I think this function could be marked as helper as well.

Reply:
In this case I needed one that would return the error from the input creation, hence I didn't use the already existing `mustCreateInput`.

I didn't mark it  as a helper because it does not call any method on `testing.T`. But there would be no harm in doing so.